public/index.html

```diff
@@ -1,54 +1,139 @@
-      <h1>Mozilla Concrete</h1>
-      <p>
-        Virtual building blocks for your virtual home!
-      </p>
-      <p>
-        OK, not really. This is just an example site that
-        sells products with Mozilla's
-        <a href="https://github.com/mozilla/payments">payment system</a>.
-      </p>
-      <div id="error"></div>
-      <div id="products">
-        <h2>Products</h2>
-        <div id="product-listing">
-          <ul>
-            <li>
-              <div class="spinner-cont">
-                <div class="spinner"></div>
-              </div>
-            </li>
-          </ul>
-        </ul>
-      </div>
-      <!--
-      <div id="your-products">
-        <h2>Purchases</h2>
-        <ul>
-          <li class="placeholder product">You haven't purchased anything yet.</li>
-        </ul>
-      </div>
-      -->
-      <div>
-        <button id="manage">Manage your account</button>
-      </div>
-    </div>
-    <footer>
-      <a target="_blank" href="https://github.com/mozilla/payments-example">source code</a>
-    </footer>
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <meta name="format-detection" content="telephone=no">
+  <meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width, height=device-height, target-densitydpi=device-dpi">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+  <link rel="stylesheet" href="css/index.css">
+  <title>Mozilla Payments</title>
+</head>
+
+<body>
+    <nav class="navbar navbar-default" role="navigation">
+  <div class="navbar-header">
+    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#nav">
+      <span class="sr-only">Toggle navigation</span>
+      <span class="icon-bar"></span>
+      <span class="icon-bar"></span>
+      <span class="icon-bar"></span>
+    </button>
+    <a class="navbar-brand" href="/">Mozilla Payments</a>
+  </div>
+  <div class="navbar-collapse collapse" id="nav">
+    <ul class="nav navbar-nav">
+        <li role="presentation"><a href="#features">Features</a></li>
+        <li role="presentation"><a href="#drop-in">Client Library</a></li>
+        <li role="presentation"><a href="#full-api">Full API</a></li>
+        <li role="presentation"><a href="#management">Management</a></li>
+        <li role="presentation"><a href="#contact">Contact Us</a></li>
+    </ul>
+  </div>
+</nav>
+
+  <div class="callout">
+      <div class="container">
+    <p><a href="https://payments.readthedocs.org">Mozilla Payments</a> is a unified
+      payments interface to Mozilla sites and properties. We provide a
+      wrapper around <a href="https://www.braintreepayments.com/">Braintree</a>.
+      Consider us a replacement for using Stripe, Braintree and others.</p>
+    <p>Work at the Mozilla Foundation or Mozilla Corporation and want to accept payments? This is for you.</p>
```

Comment:
Is there a way to rephrase this so we can focus on the products rather than the developers? Maybe something like: "Are you looking to accept payments on a Mozilla Foundation or Mozilla Corporation site?"